internal/pkg/cli/common.go

```diff
@@ -118,18 +118,20 @@ func SaveManifest(projectManifest *manifest.Manifest, logger *zap.SugaredLogger)
 	return false, nil
 }
 
-func Validate(projectState *state.State, logger *zap.SugaredLogger) error {
+func Validate(projectState *state.State, logger *zap.SugaredLogger, skipEncryptValidation bool) error {
```

Comment:
push vola validate ale s `--encrypt` flagom by to furt padalo takze som dorobil moznost preskocenia validacie encryption ak je push zadany s --encrypt flagom

Reply:
Ok, ono by to padalo najma asi s `--dry-run`, ked sa `encrypt` nevykona (ak by bola validacia az za encrypt).